test/functional/cmdLineTests/gcsuballoctests/gcsuballoctests.xml

```diff
@@ -24,10 +24,11 @@
 
 <!DOCTYPE suite SYSTEM "cmdlinetester.dtd">
 
-<suite id="J9 GC  Tests" timeout="60">
+<suite id="J9 GC Suballocator Tests" timeout="60">
  
```

Comment:
While you're here, please remove the trailing whitespace here and on line 95 and remove the extra blank line at the end of this file.

Reply:
I still see a trailing space on line 95.

Reply:
Should be good now.